UKHomeOffice/cop-react-form-renderer
# Feature/cop 10344/caseview v3

### Description
This change has been done to support changes in cop to display read-only versions of submitted forms when in CaseView. The styling has been applied to match what was outlined in the figma documentation containing the initial designs of CaseView.

https://support.cop.homeoffice.gov.uk/browse/COP-10344

### To test
Testing that the functionality works can be achieved by viewing the changes in storybook (`yarn storybook`) and looking in each of the respected sections for Check Your Answers, Form Renderer and Summary List. At the bottom there will be a read-only view which contains no change actions, or no lines.
